src/vm/vars.hpp

```diff
@@ -315,7 +315,9 @@ class REF : public OBJECTREF
 #define ObjectToOBJECTREF(obj)     (OBJECTREF(obj))
 #define OBJECTREFToObject(objref)  ((objref).operator-> ())
 #define ObjectToSTRINGREF(obj)     (STRINGREF(obj))
+#define ObjectToUTF8STRINGREF(obj) (UTF8STRINGREF(obj))
```

Comment:
This would be only needed if we were to have manually managed implementations for UTF8 string. I do not expect we are going to have any. (Except for the fast allocator - that won't need it.)